src/work-allocation/containers/task-assignment/task-assignment-container.component.ts

```diff
@@ -116,18 +116,17 @@ export class TaskAssignmentContainerComponent implements OnInit {
   }
 
   private reportSuccessAndReturn(): void {
-    const message = this.successMessage;
-    this.returnWithMessage(
-      { type: InfoMessageType.SUCCESS, message },
-      { badRequest: false }
-    );
+    this.returnWithMessage({
```

Comment:
Return this to what it was, please. `this.successMessage` comes from the data on the route so this will break another story - it shouldn't just be `InfoMessage.ASSIGNED_TASK`.

Reply:
merge issue, will resolve